app/routes/base.py

```diff
@@ -15,6 +15,9 @@ def prepare(self):
 
     def set_default_headers(self) -> None:
         self.set_header("Content-Type", "application/json")
+        self.set_header(
```

Comment:
Instead of adding it here (this is the general base method), let’s modify the allowed header on admin routes only (on line 47)